src/components/ProductListing.vue

```diff
@@ -1,22 +1,14 @@
 <template>
 <section>
+    <NavBar/>
     <main>
 		<div class="mt-28 mx-36 sm:mx-10">
             <div class="productlist grid grid-cols-4 gap-5 sm:grid-cols-2 " id="productlist">
-                <div v-for="product in productList" :key="product">
-                    <div class="border flex flex-col justify-center py-8 cursor-pointer">
-                        <div class="flex justify-center items-center">
-                            <img :src="product.thumbnail" class="h-64 w-80">
-                        </div>
-                        <div class="px-10">
-                            <p>{{product.title}}</p>
-                            <p class="font-bold text-emerald-500">Rs.{{calculateDiscoutedAmount(product)}}</p>
-                            <div class="flex">
-                                <p class="text-gray-400 mr-3 line-through text-sm">{{product.price}}</p>
-                                <p class="text-sm">-{{product.discountPercentage}}%</p>
-                            </div>
-                        </div>
-                    </div>
+                <ProductCard v-for="(product, index) in paginatedItems" :key="index" :product="product"/>
+            </div>
+            <div class="pagination flex  float-right my-3" id="pagination">
+                <div v-for= "(p, index) in page_count" :key="index">
```

Comment:
instead of naming p , create proper naming for this.so that it will be easier to read code

Reply:
done